tests/playwright/src/specs/extension-installation.spec.ts

```diff
@@ -56,6 +56,7 @@ let navBar: NavigationBar;
 const _startup = async function (): Promise<void> {
   pdRunner = new PodmanDesktopRunner();
   page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('extension-installation-e2e');
```

Comment:
This is parameterized test. You need to use naming by a parameter value. Otherwise it will get overriden or fail to save the file.

Reply:
fixed